draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
+           <vspace blankLines="1"/>
+      Description: Filtered collection URL corresponding to the label.
+           <vspace blankLines="1"/>
+           Value: A URL represented as a JSON string.
+           <vspace blankLines="1"/>
+      Mandatory: Yes.
+           </t>
+      </list></t>
+           </list></t> 
+      </section>
+   </section>
 
-        <section anchor="regex-match" title="RegexMatch">
-          <t>
-             A RegexMatch consists of a regular expression string a URI is matched against,
-             and flags describing the type of match. It is encoded as a JSON object with
-             the following properties:
-     </t>
-
-        <t><list style="empty" hangIndent="3">
-   <t><list style="hanging" hangIndent="3">
-   <t hangText="Name: regex">
-        <vspace blankLines="1"/>
-   Description: A regular expression for URI matching.
-        <vspace blankLines="1"/>
-        Value: A regular expression to match against the URI, i.e., against the path-absolute
-                 and the query string parameters <xref target="RFC3986"/>. The regular expression
-                 string MUST be compatible with <xref target="POSIX.1">POSIX</xref> Section 9
-                 Extended Regular Expressions. This regular expression MUST be evaluated in
-                 the POSIX locale (<xref target="POSIX.1">POSIX</xref> Section 7.2).
-
-              <t>Note: Because '\' has a special meaning in JSON <xref target="RFC8259"/> as the escape
-                 character within JSON strings, the regular expression character '\' MUST be escaped as '\\'.</t>
-        <vspace blankLines="1"/>
-        Mandatory: Yes.
-        </t>
-
-   <t hangText="Name: case-sensitive">
-        <vspace blankLines="1"/>
-   Description: Flag indicating whether or not case-sensitive matching should be used.
-        <vspace blankLines="1"/>
-   Value: JSON boolean. Either "true" (the matching is case-sensitive) or "false" 
-   (the matching is case insensitive).
-        <vspace blankLines="1"/>
-   Mandatory: No; default is "false", i.e., a case-insensitive match.
-        </t>
-   <t hangText="Name: match-query-string">
-        <vspace blankLines="1"/>
-   Description: Flag indicating whether to include the query part of the URI when
-                 compared against the regex.
-        <vspace blankLines="1"/>
-   Value: JSON Boolean. Either "true" (the full URI, including the query part,
-                 should be compared against the regex) or "false" (the query part of the URI
-                 should be dropped before comparison with the given regex).
-        <vspace blankLines="1"/>
-        Mandatory: No; default is "false". The query part of the URI MUST be dropped
-                 before comparison with the given regex. This makes the regular expression simpler
-      and safer for cases in which the query parameters are not relevant for the match.
+   <section title="Other CI/T Objects" anchor="other-cit-objects">
```

Comment:
This section name doesn't seem congruent with the contents of this section.  Technically, one item is an object (ObjectList), while the other two (URL type and CDN PID) appear to be just types of attributes.  It seems like the other document called this "Properties of CI/T Objects".  Maybe something like "Other CI/T Objects and Properties"?

Reply:
ACK. 
Also added a comment about this section referencing objects and properties, used by multiple CI/T objects